modules/helix/hm.nix

```
@@ -1,28 +1,41 @@
 {
+  mkTarget,
   pkgs,
-  config,
   lib,
   ...
 }:
-{
-  options.stylix.targets.helix.enable =
-    config.lib.stylix.mkEnableTarget "Helix" true;
+mkTarget {
+  name = "helix";
+  humanName = "Helix";
+
+  extraOptions.transparentBg = lib.mkOption {
+    type = lib.types.bool;
+    description = "Used to set theme even if `opacity` is null";
+    internal = true;
+    default = false;
+  };
```

Very elegant way of sharing values across `configElements`. I assume in the future when the core primitives are extended, the `opacity.<ENVIRONMENT> == 1.0` value would be stored somewhere in `config.lib.stylix`. However, this is a very future clean up that we can ignore for now.

Either way, we could standardize the `description` phrasing, sort `default` and `internal`, and generically rename the option to `transparent`:

```suggestion
  extraOptions.transparent = lib.mkEnableOption "transparent theming" // {
    default = false;
    internal = true;
  };
```